core/src/com/unciv/ui/popups/options/MultiplayerTab.kt

```diff
@@ -166,7 +166,17 @@ private fun addMultiplayerServerOptions(
         val passwordStatusTable = Table().apply {
             add(
                 if (settings.multiplayer.passwords.containsKey(settings.multiplayer.server)) {
-                    "Your userId is password secured"
+                    val userId = settings.multiplayer.userId
+                    val password = settings.multiplayer.passwords[settings.multiplayer.server] ?: ""
+                    val authStatus = UncivGame.Current.onlineMultiplayer.multiplayerServer.fileStorage()
+                        .checkAuthStatus(userId, password)
+
+                    when (authStatus) {
+                        AuthStatus.UNAUTHORIZED -> "Your current password was rejected from the server"
+                        AuthStatus.UNREGISTERED -> "Your userId does not have any password associated with it currently"
+                        AuthStatus.VERIFIED -> "Your current password has been succesfully verified"
+                        AuthStatus.UNKNOWN -> "Your authentication status could not be determined"
```

Comment:
If these are to be displayed to users, they should appear in template.properties

Reply:
So, does simply adding these lines to template.properties work? Or do I need to do something else.

Reply:
Yup, with a ` = ` at the end